homeassistant/components/deconz/light.py

```diff
@@ -204,6 +204,12 @@ def __init__(self, device: _LightDeviceT, gateway: DeconzGateway) -> None:
             if device.model_id in ("HG06467", "TS0601"):
                 self._attr_effect_list = XMAS_LIGHT_EFFECTS
 
+        # Make a best guess effort to provide a reasonable color mode as a start,
+        # this is only used as a fallback in self.color_mode to always report a
+        # supported color mode as otherwise self._attr_color_mode could be None
+        if not self.is_on and self.color_mode not in self._attr_supported_color_modes:
+            self._attr_color_mode = next(iter(self._attr_supported_color_modes))
```

Comment:
You could also initiate it to `ColorMode.UNKNOWN`, maybe that's better?

Reply:
Yes! Lets try that :)

Reply:
Thanks @emontnemery